src/main/java/seedu/address/logic/commands/AddJobCommand.java

```diff
@@ -16,28 +15,32 @@
 /**
  * Adds a job to the address book.
  */
-public class AddJobCommand extends Command {
+public class AddJobCommand extends AddCommand<Job> {
 
-    public static final String COMMAND_WORD = "add_job";
-
-    public static final String MESSAGE_USAGE =
-            COMMAND_WORD + ": Adds a job listing to the address book. " + "Parameters: " + PREFIX_NAME + "JOBNAME "
-            + PREFIX_COMPANY + "COMPANY " + PREFIX_SALARY + "SALARY " + "[" + PREFIX_REQUIREMENTS + "REQUIREMENTS] "
-            + "[" + PREFIX_DESCRIPTION + "DESCRIPTION]\n" + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "Waiter "
-            + PREFIX_COMPANY + "Starbucks " + PREFIX_SALARY + "2500 " + PREFIX_REQUIREMENTS + "Hands and feet "
+    public static final String ENTITY_WORD = "job";
+    public static final String FULL_COMMAND = COMMAND_WORD + " " + ENTITY_WORD;
```

Comment:
Is this for purposes of testing?

Reply:
Partly yes, partly I just wanted to have a intermediate variable